src/sentry/runner/formatting.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf8 -*-
+
+from __future__ import absolute_import, print_function
+
+import re
+
+# Sentry colors taken from our design system. Might not look good on all
+# termianl themes tbh
+COLORS = {
+    "white": (255, 255, 255),
+    "green": (77, 199, 13),
+    "orange": (255, 119, 56),
+    "red": (250, 71, 71),
+}
+
+SERVICE_COLORS = {
+    "server": (108, 95, 199),
+    "worker": (255, 194, 39),
+    "webpack": (61, 116, 219),
+    "cron": (255, 86, 124),
+    "relay": (250, 71, 71),
+}
+
+
+def colorize_code(pattern):
+    code = int(pattern.group("code"))
+    method = pattern.group("method")
+
+    style = (COLORS["red"], COLORS["white"])
+
+    if code >= 200 and code < 300:
+        style = (COLORS["green"], COLORS["white"])
+    if code >= 400 and code < 500:
+        style = (COLORS["orange"], COLORS["white"])
+    if code >= 500:
+        style = (COLORS["red"], COLORS["white"])
+
+    return u"{bg}{fg} {code} {reset} {method:4}".format(
+        bg="\x1b[48;2;%s;%s;%sm" % (style[0]),
+        fg="\x1b[38;2;%s;%s;%sm" % (style[1]),
+        reset="\x1b[0m",
+        code=code,
+        method=method,
+    )
+
+
+def colorize_reboot(pattern):
+    return u"{bg}{fg}[ RELOADING ]{reset} {info_fg}{info}".format(
+        bg="\x1b[48;2;%s;%s;%sm" % COLORS["red"],
+        fg="\x1b[38;2;%s;%s;%sm" % COLORS["white"],
+        info_fg="\x1b[38;2;%s;%s;%sm" % COLORS["white"],
+        reset="\x1b[0m",
+        info=pattern.group(0),
+    )
+
+
+def colorize_booted(pattern):
+    return u"{bg}{fg}[ UWSGI READY ]{reset} {info_fg}{info}".format(
+        bg="\x1b[48;2;%s;%s;%sm" % COLORS["green"],
+        fg="\x1b[38;2;%s;%s;%sm" % COLORS["white"],
+        info_fg="\x1b[38;2;%s;%s;%sm" % COLORS["white"],
+        reset="\x1b[0m",
+        info=pattern.group(0),
+    )
+
+
+def colorize_traceback(pattern):
+    return u"{bg}  {reset} {info_fg}{info}".format(
+        bg="\x1b[48;2;%s;%s;%sm" % COLORS["red"],
+        info_fg="\x1b[38;2;%s;%s;%sm" % COLORS["red"],
+        reset="\x1b[0m",
+        info=pattern.group(0),
+    )
+
+
+def monkeypatch_honcho_write(self, message):
```

Comment:
Can we somehow dispatch based on container name in here or find a way to otherwise restrict the scope of the regexes? Applying these regexes to output regardless of where the output came from is, I think, detrimental to printf-debugging.